doc/dev/osd_internals/snaps.rst

```diff
@@ -126,3 +126,84 @@ up to 8 prefixes need to be checked to determine all hobjects in a particular
 snap for a particular PG.  Upon split, the prefixes to check on the parent
 are adjusted such that only the objects remaining in the PG will be visible.
 The children will immediately have the correct mapping.
+
+clone_overlap
+-------------
+Each SnapSet attached to the head object contains the overlapping intervals
+between clone objects for optimizing space.
+The overlapping intervals are stored within the ``clone_overlap`` map, each element in the
+map stores the snap ID and the corresponding overlap with the next newest clone.
+
+See the following example using a 4 byte object:
+
+object content is: [AAAA]
+
+listnsnap output is as follows:
++------------+------------+-----------+-------------------------+
+| cloneid    | snaps      | size      | overlap                 |
++============+============+===========+=========================+
+| head       | -          | 4         |                         |
++------------+------------+-----------+-------------------------|
+
+After taking a snapshot (ID 1) and re-writing the first 2 bytes of the object,
+the clone created will overlap with the new head object in its last 2 bytes.
```

Comment:
In some places (e.g. for RGW) we write HEAD vs head, should we do so here?

Reply:
I changed to _head_ same as other references in this file. Moreover, the outputs from the examples attached use "head" (lower case).